tests/test_json_output.py

```diff
@@ -0,0 +1,51 @@
+from modules.Connection import Connection
+from BaseCollector import BaseCollector
+import pytest
+import json
+import yaml
+
+
+@pytest.fixture(scope='class')
+def bc_yaml():
+    bc_data = BaseCollector.read_rest_yaml()
+    yield bc_data
+
+
+@pytest.fixture(scope='class')
+def vmon_json():
+    with open("mockingServer/data/vmon.json", 'r') as data:
+        vmon_data = json.load(data)
+    return vmon_data
+
+
+@pytest.fixture(scope='class')
+def logging_json():
+    with open("mockingServer/data/logging.json", 'r') as data:
+        logging_data = json.load(data)
+    return logging_data
+
+
+class TestJsonOuput:
+    def test_yaml_read(self, bc_yaml):
+        with open('./rest.yaml') as yaml_file:
+            rest_yaml = yaml.safe_load(yaml_file)
+        assert bc_yaml == rest_yaml
+
+    def test_connection_get(self, setup_vcenter, bc_yaml, vmon_json):
+        key = bc_yaml['vmonservice']['api_target']
```

Comment:
```suggestion
        api_target = bc_yaml['vmonservice']['api_target']
```
Why key?

Reply:
I will add this in a separate commit, because if I commit it here, I break the lines below. Key was chosen, because it mirrors the naming convention in the Connection class. But api_target is better to understand in this context.

Reply:
done.